### PR TITLE
Changes to background done

### PR DESCRIPTION
**Pull request by deveshd2k**
Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

Changes: [Add here what changes were made in this issue and if possible provide links.]

Demo Link: [Add here the link where your changes can be seen]

Screenshots for the change:

### CODE REVIEW

**Comment by harshitjindal** [comment]
@deveshd2k Please see https://chris.beams.io/posts/git-commit/  for how to write good commit messages.

File: templates/home/home.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 {% block content %}
-<div class="jumbotron">
-  <div class="container">
-    <h1><center>Programming Club</center></h1>
-    <h3><center>UIET, Panjab University</center></h3>
+<div class="jumbotron" style="background-image: url({% static 'home/images/image_for_back.jpg' %}); background-size: 100% 100%; height:330px;">
+  <div class="container margin">
+    <h1><center><font color="#00a652">Programming Club</font></center></h1>
+    <h3><center><font color="#00a652">UIET, Panjab University</font></center></h3>
+
```
Comment (harshitjindal) [inline templates/home/home.html]: Remove this blank line and change the commit message

**Comment by divyam3897** [comment]
Fix http://picpaste.com/Screenshot_20171008_162337-fm74jbMa.png and http://picpaste.com/Screenshot_20171008_162356-d8wWyrJC.png

**Comment by divyam3897** [comment]
@deveshd2k still on it?

**Comment by deveshd2k** [comment]
@divyam3897 why the deployment is not happening?

**Comment by deveshd2k** [comment]
@divyam3897  can you review #104 please!

**Comment by sagar-kalra** [comment]
It is showing like this.

**Comment by divyam3897** [comment]
@deveshd2k still on it?

**Comment by deveshd2k** [comment]
@divyam3897 yes! fixing the footer

**Comment by deveshd2k** [comment]
@divyam3897 yes! fixing the footer.

**Pull request closed without merging**